discovery-provider/.env

```diff
@@ -12,3 +12,6 @@ audius_discprov_peer_refresh_interval=10
 audius_discprov_identity_service_url=http://docker.for.mac.localhost:7000
 audius_discprov_user_metadata_service_url=http://docker.for.mac.localhost:4000
 WAIT_HOSTS=docker.for.mac.localhost:5432,docker.for.mac.localhost:5379,docker.for.mac.localhost:6001
+# local ganache keys
+audius_delegate_owner_wallet=0x1D9c77BcfBfa66D37390BF2335f0140979a6122B
+audius_delegate_private_key=0x3873ed01bfb13621f9301487cc61326580614a5b99f3c33cf39c6f9da3a19cad
```

Comment:
nit - we should definitely not commit even local private keys

Reply:
nvm im seeing dheerajs comments now - we probably need to generate these as part of setup now

Reply:
yeah ideally we would generate these and read them dynamically. short of wiring that up thought we could do this for now, especially since the discprov sends back owner wallet. there is a disparity where wallet registered on chain for delegate != this wallet so if you check that it will fail, so we'll need to address that when we wire all this up